kevinb5617/ResuMake
# added files for info-input page

# Description
There should be a page where the user can upload/input information and content to personalize their website.

# Related Issue(s) and Pull Request(s)
[#21 ](https://github.com/kevinb5617/ResuMake/issues/21)

# Solution

Created a first draft for the information input page. It only has text input fields for "Name", "Title", and "Bio" which would be used in the About Me page. There is no functionality yet so clicking the "Submit" button will do nothing. To view this page you have to navigate to http://localhost:3000/info-input/ because there is not yet a "Next" button to navigate there.

# Steps to Test

1. `git fetch upstream`
2. `git checkout feature/info-input`
3. `cd app`
4. `npm install`
5. `npm start`
6. Navigate to http://localhost:3000/info-input/
7. There is no functionality yet, but verify that nothing is broken

# Known Issues

* Currently no functionality
* Not yet expanded to include more input fields for other pages

# Checklist
